.travis/install.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e -x
+
+# The default build branch for all repositories. This defaults to
+# TRAVIS_BRANCH unless set in the Travis build environment.
+WTSI_NPG_BUILD_BRANCH=${WTSI_NPG_BUILD_BRANCH:=$TRAVIS_BRANCH}
+
+# CPAN
+cpanm --quiet --notest Alien::Tidyp # For npg_tracking
+cpanm --quiet --notest Module::Build
+
+# WTSI NPG Perl repo dependencies
+repos=""
+for repo in perl-dnap-utilities perl-irods-wrap ml_warehouse npg_tracking npg_seq_common npg_qc npg_ml_warehouse; do
```

Comment:
I tried... npg_qc has dependency on this (see https://github.com/wtsi-npg/npg_qc/blob/devel/Build.PL) and also on modules from nag_common. In their absence building of the dependencies exists with an error. Making these dependencies optional in nog_qc does not seem right...